cmd/statusd/main.go

```diff
@@ -95,6 +99,8 @@ var logger = log.New("package", "status-go/cmd/statusd")
 func main() {
 	flag.Var(&searchTopics, "topic.search", "Topic that will be searched in discovery v5, e.g (mailserver=1,1)")
 	flag.Var(&registerTopics, "topic.register", "Topic that will be registered using discovery v5.")
+	flag.Var(&dtypes, "dtype", "Discovery type.")
```

Comment:
I would go for `discoverytype`. What values are possible?

Reply:
ethv5 and ethvousv1. defined in discovery/discovery.go

Reply:
Can we add it to `Discovery type` help text?

Reply:
removed this type. will use 2 boolean flags